ONSdigital/takeon-ui
# Feature/spp 1266/delayed refresh

# Description



Added a 6 second refresh with a loading screen when 'save and validate' button is pressed. This should mean that you don't need to refresh it more times after save button is pressed to get the validations to show up.
Story SPP-1266: https://collaborate2.ons.gov.uk/jira/browse/SPP-1266
# How to test?




Tested manually in **spp 1515** environment, can be tested manually:
Go to a survey that you can check validations for, (We tested validation on reference 12000534932 survey 023 period 201903) then enter data and press 'save and validate'. A new window should open saying 'loading' which closes after 6 seconds then refreshing the page. Validation messages for the data you entered should appear.
BDD to be tested once Fayaz has updated BDD to match new functionality.
# Links


Story SPP-1266: https://collaborate2.ons.gov.uk/jira/browse/SPP-1266
